rust/osm2lanes/src/lib.rs

```diff
@@ -148,7 +156,8 @@ impl LanePrintable for Direction {
 
 /// A map from string keys to string values. This makes copies of strings for
 /// convenience; don't use in performance sensitive contexts.
-#[derive(Clone, Deserialize)]
+// TODO: Rationale for BTreeMap iso HashMap
```

Comment:
Ah, it's just for deterministic serialization. In lots of projects I'm making sure some pipeline doesn't behaviorally change by comparing output directly, so I try and trim f64's and keep the order of output the same. It may be convenient to have that determinism here too. But maybe also there's better ways of doing it, like having a custom serde serializer that does the sorting last minute

Reply:
any rationale is fine for now, so that it can be changed confidently later when the rationale changes :)